articles/clean-and-cleaner-code.md

```diff
@@ -0,0 +1,28 @@
+---
+authorName: Milan Milanovic
+authorGithubUsername: milanm
+issue: 59
+title: Clean and cleaner code
+---
+# {{page.title}}
+
+Being a developer in these days means to develop features and fix bugs sometimes in a fast pace, which may result in a badly maintainable code. On the other hand, we also want to make the most elegant solutions to some problems. This can produce code that machine can understand but people cannot so well. Such code can become a mayor issue for a company to maintain and use in a long run. 
+Our objective should be to write maintainable, understandable, simple and readable code, and we need to make an extra effort for this to happen. Code should be easy to change and understand to our colleagues, however this is not an easy task to achieve, especially on large projects. So we need to practice writing nice code and fail, fix it and repeat again until we achieve clean code. 
```

Comment:
"So we need to practice writing nice code and fail, fix it and repeat again until we achieve clean code." is the an 
opportunity to elaborate when and how to practice

Reply:
I've added some new insights here.